nixos/modules/services/matrix/synapse.md

```diff
@@ -18,7 +18,27 @@ around Matrix.
 
 [Synapse](https://github.com/element-hq/synapse) is
 the reference homeserver implementation of Matrix from the core development
-team at matrix.org. The following configuration example will set up a
+team at matrix.org.
+
+Before you deploy synapse server, you'll need to set up a postgresql database.
+For that, please make sure that postgresql is running and the following
+SQL statements to create a user & database called `matrix-synapse` were
+executed before synapse starts up:
+
+```sql
+CREATE ROLE "matrix-synapse" WITH LOGIN PASSWORD 'synapse';
```

Comment:
matrix-synapse actually doesn't need a password, but can use socket auth out of the box. I think we want to use that.

Reply:
Good catch! Interestingly, the example below does what I'd expect it to do, i.e. being a minimal example with the DB on the same machine and socket auth.

So no idea why it was there, will remove it.